.github/workflows/node-zxf-deploy-integration.yaml

```
@@ -36,49 +36,71 @@ jobs:
           COMMIT_SHA=${{ inputs.ref }}
           COMMIT_AUTHOR="$(git show -s --format='%an' ${COMMIT_SHA})"
           COMMIT_MSG="$(git show -s --format='%s' ${COMMIT_SHA})"
-
+          
           # Print them to the console
           echo "Commit Author: $COMMIT_AUTHOR"
           echo "Commit Message: $COMMIT_MSG"
           echo "Commit SHA: $COMMIT_SHA"
-
+          
           # Expose them as outputs for the next steps
           echo "author=$COMMIT_AUTHOR" >> $GITHUB_OUTPUT
           echo "msg=$COMMIT_MSG" >> $GITHUB_OUTPUT
 
       - name: Notify Jenkins of Release (Integration)
         id: jenkins-integration
-        uses: fjogeleit/http-request-action@23ad54bcd1178fcff6a0d17538fa09de3a7f0a4d # v1.16.4
-        with:
-          url: ${{ secrets.RELEASE_JENKINS_INTEGRATION_URL }}
-          data: |
-            '{
+        run: |
+          RESPONSE=$(curl -X POST \
+            -H "Content-Type: application/json" \
+            -d '{
               "ref": "${{ inputs.ref_name }}",
               "author": "${{ steps.gather-commit-info.outputs.author }}",
               "msg": "${{ steps.gather-commit-info.outputs.msg }}",
               "sha": "${{ inputs.ref }}"
-            }'
+            }' \
+            ${{ secrets.RELEASE_JENKINS_INTEGRATION_URL }})
+          
+          echo "response<<EOF" >> $GITHUB_OUTPUT
+          echo "$RESPONSE" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+          
+          echo "Raw response from Jenkins:"
+          echo "$RESPONSE"
```

Simplify by using compact json and avoid the need for the multi-line output syntax:
```suggestion
echo "response=$(jq -c -r '.' <<<"${RESPONSE}")" >> "${GITHUB_OUTPUT}"
```